src/widgets/sections/DescriptionSection.tsx

```diff
@@ -0,0 +1,18 @@
+import { cn } from '@/shared';
+
+type Props = {
+  className?: string;
+  children: React.ReactNode;
+};
+
+export const DescriptionSection = ({
+  className,
+  children,
+  ...props
+}: React.ComponentProps<'p'> & Props) => {
+  return (
+    <p className={cn(className)} {...props}>
```

Comment:
```suggestion
    <p className={className} {...props}>
```
이러면 안 되나요?? 합칠 것도 없는데

Reply:
cn에 꽃혀서 신기해가지고 써보았습니다 허허